dependencies/common/install-boost

```diff
@@ -24,22 +24,29 @@ INSTALL_DIR=`pwd`
 PLATFORM=`uname`
 
 # vars
-BOOST_VERSION_NUMBER=1.50.0
-BOOST_VERSION=boost_1_50_0
+BOOST_VERSION_NUMBER=1.63.0
+BOOST_VERSION=boost_1_63_0
 BOOST_DIR=/opt/rstudio-tools/boost/$BOOST_VERSION
 BOOST_TAR=$BOOST_VERSION.tar.bz2
 BOOST_BUILD_DIR=boost-build
+BOOST_MODULES="algorithm asio array bind chrono circular_buffer context crc
```

Comment:
Note that some of these modules aren't actually used by us (e.g. `context`) but adding them was necessary to work around a bug in `bcp`'s dependency resolution; see e.g.

https://svn.boost.org/trac/boost/ticket/11376

The issue was marked as resolved there, but I was still seeing it -- regardless this seems to function as a workaround.